manifests/m/Microsoft/AccountLockoutStatus/1.0.0.60/Microsoft.AccountLockoutStatus.locale.en-US.yaml

```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.12.0.schema.json
+
+PackageIdentifier: Microsoft.AccountLockoutStatus
+PackageVersion: 1.0.0.60
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Account Lockout Status

```

Comment:
This doesn't match, but the publisher and product code do, so it should be fine

Reply:
Sure thing.